docs/main/create-ocp-project/README.md

```diff
@@ -23,88 +24,170 @@ TARGET_NS=sonataflow-infra
 ```
 
 # Installation
-## Persistence pre-requisites
-If persistence is enabled, you must have a PostgreSQL instance running in the cluster, in the same `namespace` as the workflows.
 
-A `secret` containing the instance credentials must exists as well. 
+## Persistence Pre-requisites
```

Comment:
Is this relevant to the workflow, or can it be removed? Does the workflow talk directly to the database?

Reply:
This shall be kept, the workflow is persisting its state to the DB directly